app/content/react/library-reference/widgets.md

```diff
@@ -155,23 +211,27 @@ Renders a `<LegendWidget />` component. The widget can display a switch to show
 - **Input**:
 
 {{% tableWrapper tab="true" %}}
-| Param         | Type           | Default       | Description    |
-| ------------- | -------------- | ------------- | -------------- |
-| props         | <code>Object</code>               |               |                |
-| [props.className] | <code>string</code>   |         | (optional) Material-UI withStyle class for styling |
+| Param                    | Type                | Default       | Description                                         |
+| ------------------------ | ------------------- | ------------- | --------------------------------------------------- |
+| props                    | <code>Object</code> |               |                                                     |
+| [props.className]        | <code>string</code> |               | (optional) Material-UI withStyle class for styling. |
+| [props.initialCollapsed] | <code>bool</code>   | `false`       | (optional) Indicates whether the widget is initially collapsed or not. |
 {{%/ tableWrapper %}}
 
 You can control the legend options through the following properties that must be added to the `layerAttributes` property for the layer in the store:
 
 {{% tableWrapper tab="true" %}}
-| Param         | Type           | Default       | Description    |
-| ------------- | -------------- | ------------- | -------------- |
-| title         | `string`       |               | Layer title    |
-| switchable    | `boolean`      | `true`        | Whether the layer can be hide/shown |
+| Param         | Type           | Default       | Description                                               |
+| ------------- | -------------- | ------------- | --------------------------------------------------------- |
+| title         | `string`       |               | Layer title                                               |
+| visible       | `boolean`      | `true`        | Indicates whether the layer is visible by default or not. |
+| opacity       | `Number`       | `1`           | Initial opacity for the layer.                            |
+| showOpacityControl | `boolean` | `true`        | Indicates whether the opacity control is shown or not.    |
+| switchable    | `boolean`      | `true`        | Indicates whether the layer can be hide/shown             |
 | legend        | `Object`       |               | Legend properties. Define an empty object `legend: {}` if you just want layer switching capabilities. |
 | legend.type   | `string`       |               | Legend type. Must be one of the types defined in the LEGEND_TYPES enum |
-| legend.attr   | `string`       |               | Attribute used for styling the layer |
-| legend.colors | `Array` or `string` |               | Array of colors (RGB arrays) or CARTO colors palette (string). Used for `LEGEND_TYPES.CATEGORY`, `LEGEND_TYPES.BINS` and `LEGEND_TYPES.CONTINUOUS_RAMP` |
+| legend.attr   | `string`       |               | Attribute used for styling the layer                      |
```

Comment:
legend.collapsed is missing.
https://github.com/CartoDB/carto-react/blob/master/packages/react-ui/src/widgets/legend/LegendWidgetUI.js#L206

Reply:
Added.